variables.tf

```diff
@@ -137,3 +137,26 @@ EOD
     azurehub_to_onprem = {}
   }
 }
+
+
+variable "vpn_aad_client_configuration" {
+  description = <<EOD
+VPN client configuration using Azure AD authorization, must match this type:
+```
+map(
+  aad_audience          string                # The client id of the Azure VPN application
+  aad_issuer            string                # The STS url for your tenant
+  aad_tenant            string                # AzureAD Tenant URL
+  address_space         list(string)          # The address space out of which IP addresses for vpn clients will be taken
+)
+```
+EOD
+  type = map(any)
```

Comment:
Can you change this to use a typed object?

Reply:
Done